src/pages/Environments.js

```diff
@@ -129,17 +129,21 @@ const Environments = () => {
 
   const refreshData = Utils.withBusyState(setLoading, async () => {
     const creator = getUser().email
-    const [newRuntimes, newDisks, galaxyDisks, newApps] = await Promise.all([
+    const [newRuntimes, newDisks, galaxyDisks, cromwellDisks, newApps] = await Promise.all([
       Ajax(signal).Runtimes.list({ creator }),
       Ajax(signal).Disks.list({ creator }),
-      Ajax(signal).Disks.list({ creator, saturnApplication: 'galaxy' }),
+      Ajax(signal).Disks.list({ creator, saturnApplication: tools.galaxy.appType }),
+      Ajax(signal).Disks.list({ creator, saturnApplication: tools.cromwell.appType }),
```

Comment:
Is there a reason to call `Disks.list` 3 times instead of once and splitting it up locally? It seems to my untrained eye that all app types would be in that first call.

Reply:
I believe the issue is that we don't know which apps or runtimes the disks returned by the first call are associated with. We could simplify this if the information returned from the server included the provisionedBy information-- that's what I was suggesting to @salonishah11.

The use case is that we wish to warn if 2 or more disks in a workspace are associated with the same app or runtime.

Reply:
This is what we get back from the server.

![image](https://user-images.githubusercontent.com/484484/143316300-b2e995b7-e6fb-4707-ac2c-9dbbf4ad6e00.png)

Reply:
Yes, I think that is what @salonishah11 is adding.